platform/x11/os_x11.cpp

```diff
@@ -116,33 +116,29 @@ void OS_X11::initialize(const VideoMode &p_desired, int p_video_driver, int p_au
 	/** XLIB INITIALIZATION **/
 	x11_display = XOpenDisplay(NULL);
 
+	char *modifiers = NULL;
 	Bool xkb_dar = False;
 	if (x11_display) {
 		XAutoRepeatOn(x11_display);
 		xkb_dar = XkbSetDetectableAutoRepeat(x11_display, True, NULL);
-	}
-
-	char *modifiers = NULL;
-
-	// Try to support IME if detectable auto-repeat is supported
 
-	if (xkb_dar == True) {
+		if (xkb_dar == True) {
 
 // Xutf8LookupString will be used later instead of XmbLookupString before
 // the multibyte sequences can be converted to unicode string.
 
 #ifdef X_HAVE_UTF8_STRING
-		modifiers = XSetLocaleModifiers("");
+			modifiers = XSetLocaleModifiers("");
 #endif
+		}
 	}
 
+	// Try to support IME if detectable auto-repeat is supported
```

Comment:
Actually that comment was at the right place, the detection of auto-repeat is that `xkb_dar` bool.

Reply:
Completely missed it when I was copy-pasting. I'll fix it. Thanks.